ess/src/API/EMBC.ESS/Resources/Cases/Evacuations/Contract.cs

```diff
@@ -113,13 +115,10 @@ public class Pet
         public string Quantity { get; set; }
     }
 
-    public class Note
+    public class Note : Case
```

Comment:
a note is not a case, it is part of the file. At the manager level, it is ok to query for a particular note or for the notes of a file, but at the repository level, querying only notes makes little sense